Frontend/UpFluxAutomation/Models/EngineerData.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace UpFluxAutomation.Helpers
+{
+    public class EngineerData
+    {
+        public string Email { get; set; }
```

Comment:
We plan to remove email from engineers api.. so keep in mind to change this in future

Reply:
Perfect I keep in mind that.